Shift3/standards-and-practices
# docs(github templates): add default title prefix and labels, use GH form schema

## Changes
_List Changes Introduced by this PR_
1. Adds default labels to issue templates
2. Adds default prefix to issue titles
3. Convert feature issue template to use GH's [form generator](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms) for easier input (works only on Public repos, so it will work on this one)

## Purpose

Make it easier to create issues, less typing.

## Approach
_How does this change address the problem?_


## Testing Steps

Will be hard to test, until merge 😓 

1. Create new issue, try the different ones
2. Create a Feature issue, a form should appear

## Learning


_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #309
